# joint_limits.yaml
default_velocity_scaling_factor: 0.2
default_acceleration_scaling_factor: 0.2

joint_limits:
  joint2_to_joint1:
    has_velocity_limits: true
    max_velocity: 1.5     # rad/s
    has_acceleration_limits: true
    max_acceleration: 3.0  # rad/s^2
  joint3_to_joint2:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 3.0
  joint4_to_joint3:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 3.0
  joint5_to_joint4:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 3.0
  joint6_to_joint5:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 3.0
  joint6output_to_joint6:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 3.0

  # Gripper (pick modest values; they just need to exist)
  gripper_base_to_gripper_left2:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 10.0
  gripper_left3_to_gripper_left1:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 10.0
  gripper_base_to_gripper_right2:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 10.0
  gripper_right3_to_gripper_right1:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 10.0
  gripper_base_to_gripper_right3:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 10.0
  gripper_controller:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 10.0

